api/envoy/extensions/filters/http/grpc_field_extraction/v3/config.proto

```diff
@@ -128,10 +129,15 @@ message FieldExtractions {
   // message Bar {
   //   string name = 1;
   // }
-  map<string, RequestFieldDisposition> request_field_extractions = 1;
+  map<string, RequestFieldValueDisposition> request_field_extractions = 1;
 }
 
-message RequestFieldDisposition {
-  // Other than all writing to the default dynamic metadata, potentially we can support
-  // 1)more destinations types like headers, 2) per RequestFieldDisposition destination.
+message RequestFieldValueDisposition {
+
+  oneof disposition {
+    // The dynamic metadata namespace. If empty, "envoy.filters.http.grpc_field_extraction" will be used by default.
+    //
+    // Unimplemented. Uses "envoy.filters.http.grpc_field_extraction" for now.
+    string namespace = 1;
```

Comment:
There's nothing in the name "namespace" that implies that this is metadata.  If we later want to add fields specifying headers, this is going to be very counter-intuitive.  How about calling this "metadata_key"?

Reply:
Updated. PTAL